src/pynws/raw_data.py

```diff
@@ -92,30 +92,30 @@ async def raw_points(
 
 
 async def raw_detailed_forecast(
-    wfo: str, x: int, y: int, websession: ClientSession, userid: str
+    wfo: str, x: int, y: int, websession: ClientSession, userid: str, units: str,
 ) -> Dict[str, Any]:
     """Return griddata response."""
     url = urls.detailed_forecast_url(wfo, x, y)
     header = get_header(userid)
-    return await _make_request(websession, url, header)
+    return await _make_request(websession, url, header, {"units": units})
```

Comment:
Can you move that into `get_header` to be consistent?

Reply:
The "units" setting isn't a header, it's passed as a parameter, so I can't really include it in the `get_header()` function. I could create a `get_params()` function to make this match the headers though.

Reply:
+1 to `get_params()`